### PR TITLE
Refactor local vault views into ca

### PR DESCRIPTION
**Pull request by HACKERALERT**

### CODE REVIEW

**Review by 9naama — APPROVED** [review]
LGTM

File: src/main/java/Main.java [hunk 1]
```diff
@@ -58,13 +66,31 @@ private static void initializeMainUi() {
...
+        final LoadLocalVaultPresenter loadLocalVaultPresenter = new LoadLocalVaultPresenter(
+                loadLocalVaultViewModel, viewManagerModel);
+        final LoadLocalVaultInteractor loadLocalVaultInteractor = new LoadLocalVaultInteractor(
+                loadLocalVaultPresenter);
+        final LoadLocalVaultController loadLocalVaultController = new LoadLocalVaultController(
+                loadLocalVaultInteractor);
+        final LoadLocalVaultView loadLocalVaultView = new LoadLocalVaultView(
+                loadLocalVaultController, loadLocalVaultViewModel, viewManagerModel);
+        views.add(loadLocalVaultView, ViewConstants.LOAD_LOCAL_VAULT_VIEW);
+        // =====================
```
Comment (bbayazit16) [inline src/main/java/Main.java:80]: We can remove // ======= since it is not required anymore.

File: src/main/java/Main.java [hunk 2]
```diff
@@ -58,13 +66,31 @@ private static void initializeMainUi() {
...
+        final CreateLocalVaultPresenter createLocalVaultPresenter = new CreateLocalVaultPresenter(
+                createLocalVaultViewModel, viewManagerModel);
+        final CreateLocalVaultInteractor createLocalVaultInteractor = new CreateLocalVaultInteractor(
+                createLocalVaultPresenter);
+        final CreateLocalVaultController createLocalVaultController = new CreateLocalVaultController(
+                createLocalVaultInteractor);
+        final CreateLocalVaultView createLocalVaultView = new CreateLocalVaultView(
+                createLocalVaultController, createLocalVaultViewModel, viewManagerModel);
+        views.add(createLocalVaultView, ViewConstants.CREATE_LOCAL_VAULT_VIEW);
+        // =====================
```
Comment (bbayazit16) [inline src/main/java/Main.java:93]: Again, we can remove // =====.

**Review by bbayazit16 — APPROVED** [review]
Looks good, thank you!

**Pull request merged**